app.py

```diff
@@ -107,6 +107,42 @@ def add_ekubo_liquidity(
     return data
 
 
+def create_stablecoin_bundle(data: pandas.DataFrame):
```

Comment:
input type is Dict[str, pandas.DataFrame]. Please specify output type as well as  Dict[str, pandas.DataFrame].

Reply:
Done! ✅